<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secDialog</class>
 <widget class="QDialog" name="secDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
     background-color: rgb(10, 79, 163); border: 5px ;
	border-radius: 0px;
	padding: 5px;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>80</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color:rgb(176, 255, 216); border: 5px ;
	border-radius: 20px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: rgb(255, 148, 129); border: 5px solid gray;
	border-radius: 20px;
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_viewcl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color:rgb(176, 255, 216); border: 5px ;
	border-radius: 20px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: rgb(255, 148, 129); border: 5px solid gray;
	border-radius: 20px;
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>View Clients</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edittrans_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>360</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color:rgb(176, 255, 216); border: 5px ;
	border-radius: 20px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: rgb(255, 148, 129); border: 5px solid gray;
	border-radius: 20px;
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Edit Tans</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>220</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color:rgb(176, 255, 216); border: 5px ;
	border-radius: 20px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: rgb(255, 148, 129); border: 5px solid gray;
	border-radius: 20px;
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Edit Cleint</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edittrans">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>290</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color:rgb(176, 255, 216); border: 5px ;
	border-radius: 20px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: rgb(255, 148, 129); border: 5px solid gray;
	border-radius: 20px;
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Add Trans</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>150</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color:rgb(176, 255, 216); border: 5px ;
	border-radius: 20px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: rgb(255, 148, 129); border: 5px solid gray;
	border-radius: 20px;
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>[+}status</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_print">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color:rgb(176, 255, 216); border: 5px ;
	border-radius: 20px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: rgb(255, 148, 129); border: 5px solid gray;
	border-radius: 20px;
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: rgb(176, 255, 216);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(45, 46, 40);
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_user">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: rgb(176, 255, 216);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(45, 46, 40);
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_pass">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: rgb(176, 255, 216);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(45, 46, 40);
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transaction</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_trans">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: rgb(176, 255, 216);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(45, 46, 40);
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit_print">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Write your recipt here to be copied:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deltrans">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>430</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color:rgb(176, 255, 216); border: 5px ;
	border-radius: 20px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: rgb(255, 148, 129); border: 5px solid gray;
	border-radius: 20px;
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Delete Trans</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
